plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray_aws.yaml

```diff
@@ -3,10 +3,11 @@ hydra:
   launcher:
     cls: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
     params:
+      # TODO set default value to be True
       stop_cluster: False
 
-      # assuming you are running from plugin root root.
-      code_dir: example
+      download_log: False
```

Comment:
1. Where is this downloading to?
2. it feels like we should also introduce the ability to sync some files back to the user local machine when the job is done.
log is just one thing, but a user may also want to download their checkpoints.